index.bs

```diff
@@ -1360,56 +1360,176 @@ devices. For example, authors wish to be able to use the web to connect with the
- :: Device ids expressed as numbers should contain sufficient entropy so as to avoid re-use or
-     potential sharing among other devices, and should not be easily guessable.
-
- : Keep the user in control
- :: Any existing device ids mapped to or stored with the current session by the user agent should
-     be cleared when users elect to "clear their cookies" (and other related settings). Above all,
-     the user should be in control of this potential tracking state and be able to reset it on
-     demand.
-
- : Hide sensitive ids behind a user permission
- :: Where device identification does not make sense to be expressed in an anonymous way, access to
-     the identifier should be limited by default. One way to limit exposure is to only surface the
-     identifier to author code <em>after</em> obtaining permission from the user.
-
- : Tie ids to the same-origin model
- :: Identifiers should be unique to the origin of the web content that is attempting to access
-     them. Web content from one origin should receive an identifier that is distinct from the
-     identifier given to web content from any other origin despite the physical device being the
-     same.
- :: Within an origin, ids may have a stable representation to web content. This ensures a
-     consistent developer experience if requesting the same device twice.
-
- : Persistable when necessary
- :: Device identifiers obtained after a complex or time-consuming device selection process may
-     prefer that author code be allowed to persist the id for use in a later session in order to
-     avoid the selection process a second time. In this case, the API should not only provide a
-     stable id during the session for the given origin, but also be able to deterministically produce
-     the same id in subsequent sessions.
+This section contains principles for consideration when designing APIs for devices.
+
+<h3 id="device-ids">Use care when exposing identifying information about devices</h3>
+
+Exposing device identifiers
+(or information about a device
+or its capabilities
+that may be used to synthesize such an identifier)
+increases the fingerprinting surface of a user agent,
+thus harming user privacy.
+Merely exposing the presence of a device
+is itself a fingerprinting concern.
+Please read the TAG's finding
+on [unsanctioned tracking](http://www.w3.org/2001/tag/doc/unsanctioned-tracking/)
+for additional details.
+
+Think carefully about
+whether it is really necessary
+to expose identifying information about the device at all.
+Please consider if your use cases could be satisfied
+by a less powerful API. [[LEAST-POWER]]
+
+Despite these concerns,
+you may find that it is necessary
+to expose identifying information about a device to the web platform.
+
+The following guidelines will help ensure
+that this is done in a consistent and privacy-friendly way:
+
+: Limit identifiable information in the id
```

Comment:
I looks like the gist of this section is
1. Don't expose identifying information about a device "such as branding, make and model numbers, etc."
2. Here's how to mint an id number/string for each device-within-a-[storage-partition](https://github.com/privacycg/storage-partitioning) safely.

I think you could say that more clearly.

Reply:
I agree. This PR didn't change this text (besides the line breaking). Maybe file a new issue for this?